src/scroll-timeline-css-parser.js

```diff
@@ -257,50 +258,53 @@ export class StyleParser {
     if (hasAnimation) {
       this.extractMatches(rule.block.contents, RegexMatcher.ANIMATION)
         .forEach(shorthand => {
+          const r = this.extractTimelineName(shorthand);
+          if(!r.timelineName)
+            return;
```

Comment:
Early return skips pushing animationName below.

Reply:
Also need to handle duration=auto below.  You could add the timeline name later, and still need to apply tweaks for duration and delays as required.